01-introduction.Rmd

```diff
@@ -158,121 +167,140 @@ Because R is open source, these developments can easily build on previous work,
+
+More recently, new packages aim at easing the creation of complex, high-quality maps with minimal code.
+The **tmap** package (released in 2014) might serve as an archetype for this kind of development [@R-tmap].
+It facilitates the user-friendly creation of thematic maps with an intuitive command-line interface  (see also [**mapmisc**](https://cran.r-project.org/package=mapmisc)) . 
 <!-- ADD THIS LATTER -->
 <!-- CITE the paper Tennekes, M. (2017) tmap: Thematic Maps in R. Forthcoming in the Journal
 of Statistical Software http://von-tijn.nl/tijn/research/presentations/tmap_user2017.pdf-->
-**tmap** provides legends, scale bars, grid lines, north arrows, optimized for mapping and integrates with the **leaflet** package (released in 2015) to offer interactive maps [@R-leaflet]. 
-Additionally, the **mapview** package was created on top of **leaflet** [@R-mapview], enabling interactive maps to be created rapidly from **sp** or **sf** objects using a range of background maps, scale bars and more.
+**tmap** provides legends, scale bars, grid lines, and north arrows optimized for mapping, and also works in harmony with the **leaflet** package (released in 2015) for an easy display of interactive maps [@R-leaflet]. 
+Similarly, the **mapview** package builds also on top of **leaflet** [@R-mapview] for interactive mapping based on **sp** or **sf** objects. **mapview** allows the access of a wide range of background maps, scale bars and more.
 
-Among all this development, most important recent evolution in R's spatial ecosystem has without doubt been support for simple features thanks to the **sf** package [@R-sf], described in Chapter \@ref(spatial-class).
+However, it is noteworthy that among all the recent developments described above, the support for simple features [**sf**; @R-sf] has been without doubt the most important evolution in R's spatial ecosystem.
+Naturally, this is the reason why we will describe **sf** in detail in Chapter \@ref(spatial-class).
 
 <!-- ## How to read this book -->
 
 ## Exercises
 
-1. Think about the terms 'GIS', 'GDS' and 'Geocomputation' described above. Which is your favorite and and why?
+1. Think about the terms 'GIS', 'GDS' and 'Geocomputation' described above. Which is your favorite, and why?
 
-1. Provide 3 reasons for using a scriptable language such as R for geocomputation instead of using an established GIS programs such as QGIS.
+2. Provide three reasons for using a scriptable language such as R for geocomputation instead of using an established GIS programs such as QGIS.
 
-1. Name two advantages and two disadvantages of using the older **sp** package compared with the new **sf** package.
+3. Name two advantages and two disadvantages of using the older **sp** package compared to the new **sf** package.
```

Comment:
Ok, I have changed that.
By the way, regarding software `mature` sounds much better than `old` :-)